mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -15,43 +20,182 @@ CLASS net/minecraft/class_57 net/minecraft/entity/Entity
 	FIELD field_1608 prevYaw F
 	FIELD field_1609 prevPitch F
 	FIELD field_1610 boundingBox Lnet/minecraft/class_25;
+	FIELD field_1611 nextStepSoundDistance I
+	FIELD field_1612 submergedInWater Z
+	FIELD field_1613 hearts I
 	FIELD field_1614 air I
 	FIELD field_1615 fireImmune Z
 	FIELD field_1616 dataTracker Lnet/minecraft/class_216;
+	FIELD field_1617 minBrightness F
+		COMMENT Set to 1 when rendering inside inventory.
+	FIELD field_1618 inChunk Z
```

Comment:
what if it was isInChunk? more wordy but possibly more intuitive? 
[Yarn for 1.20](https://maven.fabricmc.net/docs/yarn-1.20-pre4+build.2/net/minecraft/world/gen/feature/EndSpikeFeature.Spike.html#isInChunk(net.minecraft.util.math.BlockPos)) does use isInChunk but not in entity mappings so maybe theres a valid reason it isnt here